pipeline/pipeline.py

```diff
@@ -120,7 +116,8 @@ def __contains__(self, item):
     DownloadUrlFile(control),
     WgetDownload(
         wpull_args,
-        accept_on_exit_code=AcceptAny(),
+        max_tries=1,
```

Comment:
I guess they are being less recoverable over time since there is pipeline preflight checks and Wpull is starting to monitor disk space.